README.md

```diff
@@ -57,3 +57,13 @@ which is configured in [concourse/pipeline.yml](concourse/pipeline.yml).
 
 The concourse pipeline has credentials for the `govuk-forms-deployer` user in
 GOV.UK PaaS. This user has the SpaceDeveloper role, so it can `cf push` the application.
+
+## Exporting form response data
+
+Aggregated anonymous daily data can be extracted from the application for any
+specified time period using the `/data-export` route.  Output is available in
+CSV format.
+
+For example, to get data for a five day period starting 10th April 2020:
+
+     curl -XGET -H "Accept: text/csv" "<path-to-service>/data-export?start_date=2020-04-08&end_date=2020-04-15"
```

Comment:
Thanks for adding this 🎉. The url needs `.csv` added to it to get the CSV download. 
i.e. `/data-export.csv?start_date=2020-04-08&end_date=2020-04-15"`

Reply:
I didn't know Rails let you do this.  That means I can remove the accept header too.